saleor/graphql/core/types/common.py

```diff
@@ -209,6 +211,14 @@ class MetadataError(Error):
     code = MetadataErrorCode(description="The error code.", required=True)
 
 
+class OrderGrantRefundCreateError(Error):
```

Comment:
For new mutation I'm adding such classes in the mutation file, e.g. this one would go to `order_grant_refund_create.py`, as it's used only there. And the `OrderGrandRefundCreateErrorCode` could go to `order/graphql/enums.py`.